helix-core/src/main/java/org/apache/helix/task/TaskStateModel.java

```diff
@@ -283,6 +292,38 @@ public void reset() {
     }
   }
 
+  /**
+   * Loads Task and TaskFactory classes for command input
+   */
+  private boolean loadNewTask(String command) {
+    try {
+      // Read ZNRecord containing task definition information.
+      ZNRecord taskConfig = _manager.getHelixDataAccessor().getBaseDataAccessor()
+          .get("/" + _manager.getClusterName() + "/TASK_DEFINITION",
+              null, AccessOption.THROW_EXCEPTION_IFNOTEXIST);
```

Comment:
Instead of doing this, you could check whether taskConfig is null.

Reply:
Thank you for the comments, Hunter. I'll use them all in the next commit.